apps/nextjs-website/src/app/auth/login/page.tsx

```diff
@@ -10,33 +10,37 @@ import ConfirmSignUp from '@/components/organisms/Auth/ConfirmSignUp';
 import { useRouter, useSearchParams } from 'next/navigation';
 import PageBackgroundWrapper from '@/components/atoms/PageBackgroundWrapper/PageBackgroundWrapper';
 import { SignInOpts } from '@aws-amplify/auth/lib/types';
+import AuthStatus from '@/components/organisms/Auth/AuthStatus';
 
 const Login = () => {
   const router = useRouter();
   const [logInStep, setLogInStep] = useState(LoginSteps.LOG_IN);
   const [user, setUser] = useState(null);
   const [username, setUsername] = useState('');
```

Comment:
in apps/nextjs-website/src/components/organisms/Auth/LoginForm.tsx the state of the username and password is handled in a single object, I'd do the same thing here to reduce the amount of uses of useState

```suggestion
    const [formData, setFormData] = useState({
    username: '',
    password: '',
  });
```